packages/conf-xen/conf-xen.1/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+maintainer: "hannes@mehnert.org"
+authors: [ "hannes@mehnert.org" ]
+homepage: "https://xenproject.org/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: [ "GPLv2" "LGPL3" ]
```

Comment:
@hannesm I'm reviewing licenses used in xapi, and I see most of the fiels are licenses with `LGPL-2.1-only`, `LGPL-2.1-or-later` in xenctrl and the other C client libraries in xen, do you re member why you added only both these licenses? (there are other licenses as all files have define their own licenses and all files must be reviewed individually, unfortunately)

Reply:
It is likely I did not check any license by myself, but just copied the licenses from the FreeBSD port (https://www.freshports.org/sysutils/xen-tools/).

Reply:
That makes sense, thanks!